test/fixtures/mixed-global-scoped.out.js

```diff
@@ -1,8 +1,8 @@
 import _JSXStyle from 'styled-jsx/style';
-const Test = () => <_JSXStyle styleId={188072295} css={"p { color: red }"} />;
+const Test = () => <_JSXStyle styleId={188072295} css={" p {color: red }"} />;
 
 export default (() => <div data-jsx={793889750}>
     <p data-jsx={793889750}>test</p>
-    <_JSXStyle styleId={3149549172} css={"body { background: red }"} />
+    <_JSXStyle styleId={3149549172} css={" body {background: red }"} />
```

Comment:
@nikvm actually you might want to just add to [L120](https://github.com/zeit/styled-jsx/blob/master/lib/style-transform.js#L120)

```js
if (prefix.length === 0) {
    return blob.trim()
}
```

since it's using an 0 length string for global allowing the middleware to only handle non-global selectors.